modules/core/src/lib/data/data-manager.js

```diff
@@ -0,0 +1,129 @@
+/* global requestAnimationFrame */
+import DataResource from './data-resource';
```

Comment:
Probably too early to settle naming. It is pretty good, but a bit inconsistent. A control question could be what we will we be calling these things in the docs? "data", "resources", "data resources", "data sources" etc.

Possibilities 
- `ResourceManager` and `Resource`
- `DataManager` and `Data`
- `DataResourceManager` and `DataResource`? And all them "data resources" in the docs?